### PR TITLE
create config toml at `DCOS_CONFIG` if doesn't exist

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by tamarrow** [comment]
#449

File: dcos/util.py [hunk 1]
```diff
@@ -130,6 +130,7 @@ def ensure_file_exists(path):
 
     if not os.path.exists(path):
         try:
+            ensure_dir_exists(os.path.dirname(path))
```
Comment (jsancio) [inline dcos/util.py]: We should only create directory for DCOS_CONFIG when the value is the default. We should not create the directories when the user has overwritten their DCOS_CONFIG.

**Comment by tamarrow** [comment]
@jsancio updated

**Comment by jsancio** [comment]
:feet:

**Pull request merged**